prov/efa/src/efa.h

```diff
@@ -231,16 +231,6 @@ struct efa_ep {
 	bool			util_ep_initialized;
 };
 
-struct efa_send_wr {
```

Comment:
Why move the definitinion? it is still used by dgram endpoint, and should remain here.